assets/js/modules/reader-revenue-manager/datastore/settings.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * `modules/reader-revenue-manager` data store: settings tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import {
+	createTestRegistry,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+
+describe( 'modules/reader-revenue-manager settings', () => {
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	describe( 'validateCanSubmitChanges', () => {
+		it( 'it validates', () => {
+			//  TODO: Implement tests as part of #8793.
```

Comment:
Why are the tests being left for #8793, when `validateCanSubmitChanges()` is implemented in this issue?

Reply:
We still need to implement the settings datastore. Hence, we don't have the selectors and actions. It will be implemented In [#8793](https://github.com/google/site-kit-wp/issues/8793).

Reply:
Ah, cool thanks for clarifying.